<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPSLoginClass</class>
 <widget class="QMainWindow" name="IPSLoginClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IPSLogin</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;minw&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="loginarea">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>35</y>
      <width>740</width>
      <height>1010</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#FFFFFF;
border-radius:10px;padding:2px 4px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="seot">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>196</y>
      <width>297</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/res/login/seot.png);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tips">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>516</y>
      <width>380</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="right">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>850</y>
      <width>288</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:24px</string>
    </property>
    <property name="text">
     <string>ICT6040 安检CT 登录页面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>562</y>
      <width>576</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ide">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>336</y>
      <width>576</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwe">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>434</y>
      <width>576</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_left_circle">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>876</y>
      <width>9</width>
      <height>9</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/res/login/circle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_right_circle">
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>876</y>
      <width>9</width>
      <height>9</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/res/login/circle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_userid">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>354</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/res/login/userid.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pwid">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>452</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/res/login/userpw.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>352</y>
      <width>1</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CDA8F0</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>450</y>
      <width>1</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CDA8F0</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>0</y>
      <width>1355</width>
      <height>958</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_circle1">
    <property name="geometry">
     <rect>
      <x>1311</x>
      <y>991</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/res/login/circle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_circle2">
    <property name="geometry">
     <rect>
      <x>1341</x>
      <y>991</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/res/login/emcircle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_circle3">
    <property name="geometry">
     <rect>
      <x>1371</x>
      <y>991</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/res/login/emcircle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>0</y>
      <width>1355</width>
      <height>958</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>label</zorder>
   <zorder>loginarea</zorder>
   <zorder>seot</zorder>
   <zorder>tips</zorder>
   <zorder>right</zorder>
   <zorder>login</zorder>
   <zorder>ide</zorder>
   <zorder>pwe</zorder>
   <zorder>label_left_circle</zorder>
   <zorder>label_right_circle</zorder>
   <zorder>label_userid</zorder>
   <zorder>label_pwid</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>label_circle1</zorder>
   <zorder>label_circle2</zorder>
   <zorder>label_circle3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
